src/dev/src/app/login/login.demo.html

```diff
@@ -12,28 +12,32 @@ <h3 class="welcome">Welcome to</h3>
         Company Product Name
         <h5 class="hint">Use your Company ID to sign in or create one now</h5>
       </section>
-      <form class="login-group">
+      <form class="login-group" #loginForm="ngForm">
         <clr-select-container>
+          <label>User Role</label>
           <select clrSelect name="type" [(ngModel)]="form.type">
             <option value="local">Local Users</option>
             <option value="admin">Administrator</option>
           </select>
         </clr-select-container>
         <clr-input-container>
-          <input type="text" name="username" clrInput placeholder="Username" [(ngModel)]="form.username"/>
+          <label>Username</label>
```

Comment:
Can we make these labels screenreader only?

Adding the labels is a breaking change for the visual design. The labels and placeholders are now redundant (visually).

If we can present the labels as `sr-only` items, then I think we can move forward.

If not, I think we need to involve visual design and likely Jehad as well because this is a visual breaking change that will affect all of our applications.

And we are planning to drop it in a minor and backport it...

Reply:
Well...since this is fully on the app side to implement we can let them decide can't we? Its a breaking visual change but its opt in.

Reply:
I don't think it will be approached as "opt in" though. Applications are going to go forward with this new look (including duplicate text in labels and placeholders).

If this is our recommendation, I think it needs a pass through UX. I don't think we can push this out as-is.